pkg/pod/v1alpha1/kubeclient.go

```diff
@@ -0,0 +1,96 @@
+package v1alpha1
+
+import (
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	kclient "github.com/openebs/maya/pkg/client/k8s/v1alpha1"
```

Comment:
Please start making use of `pkg/kubernetes/client/v1alpha1` instead of `pkg/client/k8s/v1alpha1`

Reply:
changed